src/components/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import './LoginPage.scss';
+
+const LoginPage: React.FC = (): JSX.Element => {
+	const { t } = useTranslation('LoginPage');
+
+	return (
+		<div className="login__section--inner">
+			<div className="row row-cols-md-2 row-cols-1" style={{ alignItems: 'center' }}>
+				<div className="col">
+					<div className="account__login">
+						<div className="account__login--header mb-25">
+							<h2 className="account__login--header__title h3 mb-10">{t('login')}</h2>
+							<p className="account__login--header__desc">{t('header_desc')}</p>
+						</div>
+						<div className="account__login--inner">
+							<input
+								className="account__login--input"
+								placeholder={t('placeholder_email')}
+								type="text"
+							/>
+							<input
+								className="account__login--input"
+								placeholder={t('placeholder_password')}
+								type="password"
+							/>
+							<div className="account__login--remember__forgot mb-15 d-flex justify-content-between align-items-center">
+								<div
+									className="account__login--remember position__relative"
+									style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}
```

Comment:
use bootstrap  `d-flex justify-content-between align-items-center` or
include in scss